Core/Core/Conferences/ConferenceListViewController.swift

```diff
@@ -0,0 +1,168 @@
+        refreshControl.addTarget(self, action: #selector(refresh), for: .primaryActionTriggered)
+        tableView.refreshControl = refreshControl
+        tableView.registerHeaderFooterView(SectionHeaderView.self)
+        tableView.separatorColor = .named(.borderMedium)
+
+        colors.refresh()
+        if context.contextType == .course {
+            course.refresh()
+        } else {
+            group.refresh()
+        }
+        conferences.refresh()
+    }
+
+    public override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        if let selected = tableView.indexPathForSelectedRow {
+            tableView.deselectRow(at: selected, animated: true)
+        }
+    }
+
+    func updateNavBar() {
+        guard
+            let name = context.contextType == .course ? course.first?.name : group.first?.name,
+            let color = context.contextType == .course ? course.first?.color : group.first?.color
+        else {
+            return
+        }
+        view.tintColor = color
+        spinnerView.color = color
+        refreshControl.color = color
+        updateNavBar(subtitle: name, color: color)
+    }
+
+    func update() {
+        spinnerView.isHidden = !conferences.pending || !conferences.isEmpty || conferences.error != nil || refreshControl.isRefreshing
+        emptyView.isHidden = conferences.pending || !conferences.isEmpty || conferences.error != nil
+        errorView.isHidden = conferences.error == nil
+        tableView.reloadData()
+    }
+
+    @objc func refresh() {
+        conferences.refresh(force: true) { [weak self] _ in
+            self?.refreshControl.endRefreshing()
+        }
+    }
+}
+
+extension ConferenceListViewController: UITableViewDataSource, UITableViewDelegate {
+    public func numberOfSections(in tableView: UITableView) -> Int {
+        return conferences.sections?.count ?? 0
+    }
+
+    public func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        var count = conferences.sections?[section].numberOfObjects ?? 0
+        if conferences.hasNextPage, conferences.sections?.count == section + 1 {
+            count += 1
+        }
+        return count
+    }
+
+    public func tableView(_ tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
+        let view = tableView.dequeueHeaderFooter(SectionHeaderView.self)
+        view.titleLabel?.text = conferences[IndexPath(row: 0, section: section)]?.isConcluded == true
+            ? NSLocalizedString("Concluded Conferences", bundle: .core, comment: "")
+            : NSLocalizedString("New Conferences", bundle: .core, comment: "")
+        return view
+    }
+
+    public func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        if conferences.hasNextPage && indexPath.row == conferences.sections?[indexPath.section].numberOfObjects {
+            conferences.getNextPage()
```

Comment:
With the paging and the scope order, i was able to see conferences come in and then i scrolled down and as it loaded more, there were conferences that were placed at the top of the list that were never seen until i scrolled back up which could be a little odd to the user.

Reply:
Yeah, Peyton & Dave wanted the active conferences to be at the top, so that does make paging weird. I hope in practice that there is not a massive list of newer conferences than then one that is currently active.